python_modules/libraries/dagster-postgres/dagster_postgres_tests/compat_tests/test_back_compat.py

```diff
@@ -1213,3 +1213,35 @@ def test_add_bulk_actions_job_name_column(hostname, conn_string):
                     )[0].backfill_id
                     == after_migration.backfill_id
                 )
+
+
+def test_add_run_tags_run_id_idx(hostname, conn_string):
+    _reconstruct_from_file(
+        hostname,
+        conn_string,
+        file_relative_path(
+            __file__,
+            "snapshot_1_9_3_add_run_tags_run_id_idx/postgres/pg_dump.txt",
+        ),
+    )
+
+    with tempfile.TemporaryDirectory() as tempdir:
+        with open(
+            file_relative_path(__file__, "dagster.yaml"), "r", encoding="utf8"
+        ) as template_fd:
+            with open(os.path.join(tempdir, "dagster.yaml"), "w", encoding="utf8") as target_fd:
+                template = template_fd.read().format(hostname=hostname)
+                target_fd.write(template)
+
+        with DagsterInstance.from_config(tempdir) as instance:
+            # Before migration
+            assert "run_tags" in get_tables(instance)
+            assert "idx_run_tags" in get_indexes(instance, "run_tags")
+            assert "idx_run_tags_run_id" not in get_indexes(instance, "run_tags")
+
+            # After upgrade
+            instance.upgrade()
+
+            assert "run_tags" in get_tables(instance)
+            assert "idx_run_tags" not in get_indexes(instance, "run_tags")
+            assert "idx_run_tags_run_id" in get_indexes(instance, "run_tags")
```

Comment:
Even though it is recommended in alembic/README.md to run downgrade migration, I couldn't find any example for postgres. Is that okay?

Reply:
This looks fine to me.